packages/paste-website/src/components/site-wrapper/sidebar/SidebarNavigation.tsx

```diff
@@ -124,7 +124,7 @@ const SidebarNavigation: React.FC<SidebarNavigationProps> = () => {
   const [tokensOpen, setTokensOpen] = React.useState(getCurrentPathname().startsWith(SidebarCategoryRoutes.TOKENS));
 
   return (
-    <SiteNav>
+    <SiteNav tabIndex={0} role="navigation" aria-label="Main">
```

Comment:
Again, no need for the role here, it is a `<nav />` element that comes with an implicit `navigation` role.

The label is a nice addition.

I am cautious of the focusable nav element here though. Ordinarily non-interactive elements shouldn't be focusable, even if they scroll. I'm OK with the main as we'll use it as a jump point when we have a "skip to main content" skip link, but I don't think you need it on the nav. The panel will scroll as you tab through the nav items